packages/material-ui/src/Unstable_TrapFocus/Unstable_TrapFocus.js

```diff
@@ -23,6 +23,8 @@ function Unstable_TrapFocus(props) {
   const sentinelStart = React.useRef(null);
   const sentinelEnd = React.useRef(null);
   const nodeToRestore = React.useRef();
+  const insideReactTreeRef = React.useRef(false);
+  const insideReactTreeEventTarget = React.useRef();
```

Comment:
```suggestion
  const reactFocusEventTarget = React.useRef();
```

We wouldn't be able to track react events outside of this tree anyway. The new name makes it clear what kind of events we're tracking.